jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -1,9 +1,14 @@
 #!/bin/bash
 set -eu -o pipefail
 
+declare -r RESULT_SUCCESS=0
+declare -r RESULT_ERROR_INFRA=1
```

Comment:
Not sure here, but shouldn't we use reserved bash exit codes?

Reply:
What do you mean by "reserved bash exit codes"?

Reply:
https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
As we have an exit handler in place which ensures that bash-internal exit codes are translated to `1`, I think we can stick to the exit code definition proposed.

However, as I [commented](https://github.com/SAP/stewardci-jenkinsfilerunner-image/pull/97#discussion_r1084905970), the exit handler should be installed at the very beginning of the script.